giraffe/src/components/geo/processing/tableProcessing.ts

```diff
@@ -26,12 +27,26 @@ export const GEO_HASH_COLUMN = 's2_cell_id'
 export const preprocessData = (
```

Comment:
can you add some unit tests for this function? basically test that it returns the type of table you expect given all the inputs and outputs?